Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.cxx

```diff
@@ -469,6 +471,60 @@ void qMRMLSubjectHierarchyComboBox::setAlignPopupVertically(bool align)
   d->AlignPopupVertically = align;
 }
 
+//--------------------------------------------------------------------------
+bool qMRMLSubjectHierarchyComboBox::noneItemEnabled()const
+{
+  if (!this->model())
+    {
+    return false;
+    }
+  return this->model()->noneItemEnabled();
+}
+
+//--------------------------------------------------------------------------
+void qMRMLSubjectHierarchyComboBox::setNoneItemEnabled(bool enable)
+{
+  if (!this->model())
+    {
+    return;
+    }
+  this->model()->setNoneItemEnabled(enable);
+}
+
+//--------------------------------------------------------------------------
+QString qMRMLSubjectHierarchyComboBox::noneItemText()const
+{
+  if (!this->model())
+    {
+    return QString();
+    }
+  return this->model()->noneItemText();
+}
+
+//--------------------------------------------------------------------------
+void qMRMLSubjectHierarchyComboBox::setNoneItemText(const QString& displayName)
```

Comment:
It would be better to mirror qMRMLNodeComboBox API -> rename to `setNoneDisplay`.